saul-core/src/test/scala/ssvm/randomDataApp.scala

```diff
@@ -0,0 +1,125 @@
+package ssvm
```

Comment:
I can change the name though I have made this example as a base test for ssvm in SL.